### PR TITLE
Update Readme: fix name, set IRC and Matrix links

### PR DESCRIPTION
**Pull request by codeurimpulsif**
- Remove confusing "Exodus Privacy" term (Exodus is the app, Exodus-Privacy is the organization)
- Add a "Getting in touch with us" section with IRC and Matrix links

### CODE REVIEW

**Review by jfoucry — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 <a href="https://f-droid.org/packages/org.eu.exodus_privacy.exodusprivacy/" target="_blank">
 <img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png" alt="Get it on F-Droid" height="80"/></a>
 <a href="https://play.google.com/store/apps/details?id=org.eu.exodus_privacy.exodusprivacy" target="_blank">
 <img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png" alt="Get it on Google Play" height="80"/></a>
+
+## Getting in touch with us
+
+If you want to help us improve this application, you can:
+
+- Use [the issues](https://github.com/Exodus-Privacy/exodus/issues)
```
Comment (pnu-s) [inline README.md:18]:
```suggestion
- Use [the issues](https://github.com/Exodus-Privacy/exodus-android-app/issues)
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+- Join us on our [IRC channel #exodus-android-app on Libera.chat](https://web.libera.chat/?nick=webguest?#exodus-android-app)
+- Join us on our [Matrix channel #exodus-android-app:matrix.org](https://matrix.to/#/#exodus-android-app:matrix.org)
```
Comment (pnu-s) [inline README.md:20]: Can we somehow make it more clear that this is the same channel, but a different way to join?

**Review by pnu-s — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (codeurimpulsif) [reply README.md]: Better?

Comment (pnu-s) [reply README.md]: Yes, thanks!

**Review by pnu-s — APPROVED** [review]
LGTM

**Pull request merged**